agent/util-scripts/pbench-tool-meister-client

```diff
@@ -47,21 +47,26 @@ def main(argv):
     try:
         group = argv[1]
     except IndexError:
-        raise Exception("Missing group argument")
+        logger.error("Missing group argument")
```

Comment:
Just to confirm...you want the diagnostics sent to the _log_ and not to the user?  (Or, is it the case that, in this context, the log is `stderr`?)

Reply:
I believe in this case the log is `stderr`.  See https://docs.python.org/3/library/logging.handlers.html#logging.StreamHandler.